JJMasterData.FormElementUpdater/Models/GridActions.cs

```diff
@@ -12,13 +12,13 @@ namespace JJMasterData.Core.DataDictionary;
 public class GridActions
 {
     [DataMember(Name = "editAction")]
-    public EditAction EditAction { get; private set; }
```

Comment:
These are not the production implementation, thery are used in FormElementUpdater